pkg/agent/types/annotations.go

```diff
@@ -32,4 +32,7 @@ const (
 
 	// ServiceLoadBalancerModeAnnotationKey is the key of the Service annotation that specifies the Service's load balancer mode.
 	ServiceLoadBalancerModeAnnotationKey string = "service.antrea.io/load-balancer-mode"
+
+	// L7FlowExportAnnotationKey is the key of the L7 network flow export annotation that enables L7 network flow export for annotated Pod or Namespace based on the value of annotation which is direction of traffic.
+	L7FlowExportAnnotationKey string = "visibility.antrea.io/enable-l7"
```

Comment:
How about "visibility.antrea.io/l7-flows" or "visibility.antrea.io/l7-export"?

Reply:
keeping intact with the feature, I am inclined more towards l7-export